src/main/java/com/questhelper/helpers/quests/perilousmoon/PerilousMoon.java

```diff
@@ -0,0 +1,702 @@
+			"Talk to Jessamine in the far north end of Cam Torum.");
+		talkToJessamine.addDialogStep("Can I be of any help with the dig?");
+		enterNeypotzli = new ObjectStep(this, ObjectID.ENTRANCE_51375, new WorldPoint(1439, 9600, 1),
+			"Enter the Neypotzli entrance in the far north of the cavern.");
+		// Entered, varp 4066 8174-> 40942
+		talkToAttalaInNey = new NpcStep(this, NpcID.ATTALA, new WorldPoint(1433, 9632, 1),
+			"Talk to Attala near the entrance.");
+		talkToZumaInNey = new NpcStep(this, NpcID.ZUMA_12860, new WorldPoint(1447, 9632, 1),
+		"Talk to Zuma near the entrance.");
+		talkToJessamineInNey = new NpcStep(this, NpcID.JESSAMINE_12865, new WorldPoint(1440, 9639, 1),
+			"Talk to Jessamine near the entrance.");
+
+		takeBuildingSupplies = new ObjectStep(this, ObjectID.BUILDING_SUPPLIES_50857, new WorldPoint(1436, 9637, 1),
+			"Take some building supplies from the crates near Jessamine.", emptyInventory5);
+
+		enterPrison = new ObjectStep(this, ObjectID.ENTRANCE_51376, new WorldPoint(1421, 9613, 1),
+			"Enter the south-west entrance.", buildingSupplies, hammer, saw);
+		// Path to prison camp
+		List<WorldPoint> prisonPath = new ArrayList<>(List.of(
+			new WorldPoint(1388, 9573, 0),
+			new WorldPoint(1388, 9561, 0),
+			new WorldPoint(1376, 9561, 0),
+			new WorldPoint(1376, 9546, 0),
+			new WorldPoint(1357, 9546, 0),
+			new WorldPoint(1357, 9580, 0),
+			new WorldPoint(1350, 9581, 0)
+		));
+
+		// TODO: Work out build spot ID
+		// 50037?
+		buildPrisonCamp = new ObjectStep(this, NullObjectID.NULL_53007, new WorldPoint(1349, 9580, 0),
+			"Make your way to the camp spot, marked on the minimap with a cooking range icon.", buildingSupplies, hammer, saw);
+		((ObjectStep) buildPrisonCamp).setLinePoints(prisonPath);
+		leavePrison = new ObjectStep(this, ObjectID.ENTRANCE_51375, new WorldPoint(1388, 9576, 0),
+			"Leave the area back to the antechamber.");
+		((ObjectStep) leavePrison).setLinePoints(Lists.reverse(prisonPath));
+
+		enterEarthbound = new ObjectStep(this, ObjectID.ENTRANCE_51376, new WorldPoint(1421, 9650, 1),
+			"Enter the north-west entrance.", buildingSupplies, hammer, saw);
+
+		List<WorldPoint> earthPath = new ArrayList<>(List.of(
+			new WorldPoint(1402, 9717, 0),
+			new WorldPoint(1402, 9722, 0),
+			new WorldPoint(1389, 9722, 0),
+			new WorldPoint(1389, 9727, 0),
+			new WorldPoint(1375, 9727, 0),
+			new WorldPoint(1375, 9720, 0),
+			new WorldPoint(1367, 9712, 0),
+			new WorldPoint(1367, 9697, 0),
+			new WorldPoint(1375, 9697, 0),
+			new WorldPoint(1375, 9709, 0)
+		));
+		buildEarthCamp = new ObjectStep(this, NullObjectID.NULL_53007, new WorldPoint(1374, 9710, 0),
+			"Make your way to the camp spot, marked on the minimap with a cooking range icon.", buildingSupplies, hammer, saw);
+		((ObjectStep) buildEarthCamp).setLinePoints(earthPath);
+		leaveEarth = new ObjectStep(this, ObjectID.ENTRANCE_51375, new WorldPoint(1404, 9716, 0),
+			"Leave the area back to the antechamber.");
```

Comment:
nit/unimportant/taste: simplify
```suggestion
			"Return to the antechamber.");
```